File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/InitialGpsEventHandler.java
package com.mapbox.services.android.navigation.v5.navigation;

import static com.mapbox.services.android.navigation.v5.navigation.NavigationMetricsWrapper.sendInitialGpsEvent;

class InitialGpsEventHandler {

  void send(double elapsedTime, String sessionId) {
    sendInitialGpsEvent(elapsedTime, sessionId);
  }
}

File: libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/InitialGpsEventHandler.kt
package com.mapbox.services.android.navigation.v5.navigation

import com.mapbox.services.android.navigation.v5.navigation.NavigationMetricsWrapper.sendInitialGpsEvent

internal class InitialGpsEventHandler {

    fun send(
        elapsedTime: Double,
        sessionId: String,
        metadata: NavigationPerformanceMetadata?
    ) {
        sendInitialGpsEvent(elapsedTime, sessionId, metadata)
    }
}
